.github/workflows/ci.yml

```diff
@@ -27,6 +27,8 @@ jobs:
           echo SCHEMA=orb >> .env
           echo PORT=8080 >> .env
           echo API_KEY_SYNC=api_key_test >> .env
+          echo SENTRY_DSN='https://11f1111f11c11c111a11dab1c1111fa2@o111111.ingest.us.sentry.io/1111111111111111' >> .env
```

Comment:
These two should be optional and not needed for CI testing

Reply:
Done